### PR TITLE
[Editor] do not update highlight line by default

### PR DESCRIPTION
**Pull request by jasonLaster**
Fixes Issue: #6588

### Summary of Changes

Previously we were updating the highlight line if two conditions were not met. Now, we update it if those conditions are not met and a some others are. This will prevent spurious re-renders.

### CODE REVIEW

File: src/components/Editor/HighlightLine.js [hunk 1]
```diff
@@ -66,7 +66,14 @@ export class HighlightLine extends Component<Props> {
       return false;
     }
 
-    return true;
+    if (
+      editorLine !== this.previousEditorLine ||
+      this.props.selectedSource !== selectedSource
```
Comment (bomsy) [inline src/components/Editor/HighlightLine.js]: having the `this.props.selectedSource !== selectedSource` check in, seems to make the highlight flash a really quick, that sometimes you might not see the line highlighted.

**Comment by jasonLaster** [comment]
We might not need this once #6601 lands

**Comment by bomsy** [comment]
is this still needed?

**Comment by jasonLaster** [comment]
I don't think so!

**Pull request closed without merging**